src/models/session/dissolutionSession.model.ts

```diff
@@ -20,4 +20,5 @@ export default interface DissolutionSession {
   isMultiDirector?: boolean
   isApplicantADirector?: boolean
   paymentStateUUID?: string
+  applicationPaid?: boolean
```

Comment:
`isApplicationAlreadyPaid` would be a better name looking at other booleans in this model